### ISSUE TITLE
Cannot auto confirm the social sign in prompt on tunnel run

### ISSUE THREAD

**Issue by Benehiko** [ory/cli#219]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/cli/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/cli/blob/master/CONTRIBUTING.md).
- [X] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [X] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

The Ory tunnel will now prompt the user about using the social sign in option like so:

```shell
ory tunnel http://localhost:3000 --project <slug> --dev
To support complex flows such as Social Sign In, the Ory CLI can configure your project automatically. To do so, you need to be signed in. Do you want to sign in? [y/n]:
```

When using the flag `--yes` or `-y` on the command it doesn't confirm it and continue.

### Reproducing the bug

Run `ory tunnel` for the first time and the prompt will appear. Try skip it by adding the `--yes` flag:

```shell
ory tunnel http://localhost:3000 --project optimistic-turing-6cxufsm48m --dev --yes
To support complex flows such as Social Sign In, the Ory CLI can configure your project automatically. To do so, you need to be signed in. Do you want to sign in? [y/n]:
```

### Version

v0.1.41

### On which operating system are you observing this issue?

Linux

### In which environment are you deploying?

Ory Cloud

### Additional Context

Reported on slack:
https://ory-community.slack.com/archives/C010F7Z4XM1/p1663858573933829

Right now a workaround could be for the user to pipe in the confirmation themselves:

```shell
sh -c "yes | ory tunnel http://localhost:3000/ --project <project-slug> --dev"
```

**Comment by aeneasr** [comment]
`ory tunnel http://localhost:3000/ --project <project-slug> --quiet` will do the trick.

**Comment by Benehiko** [comment]
Okay nice. Shouldn't we maybe update the cli to be more descriptive? 

For instance when running:

```shell
➜  ~ ory tunnel -h
...

Flags:
      --allowed-cors-origins strings   A list of allowed CORS origins. Wildcards are allowed.
  -c, --config string                  Path to the Ory Cloud configuration file.
      --cookie-domain string           Set a dedicated cookie domain.
      --debug                          Use this flag to debug, for example, CORS requests.
      --default-redirect-url string    Set the URL to redirect to per default after e.g. login or account creation.
      --dev                            Use this flag when developing locally.
  -h, --help                           help for tunnel
      --port int                       The port the proxy should listen on. (default 4000)
      --project string                 The slug of your Ory Cloud Project.
  -q, --quiet                          Be quiet with output printing.
  -y, --yes                            Confirm all dialogs with yes.
```

We can se the `-q` or `--quiet` flag there, but it just says it won't be printing output. In this case I would assume the `-y` or `--yes` flag to be the one confirming if I want social sign in for example.

**Comment by aeneasr** [comment]
Yeah true, we should update the description a bit. Something like "Surpresses all dialogues and answers with the default". Contribs welcomed :)
